audio/faad2/Portfile

```diff
@@ -1,10 +1,12 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem      1.0
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        knik0 faad2 2_10_0
```

Comment:
What version number is MacPorts going to get from this given the underscores? Ideally we want it to think it's 2.10.0 and not 2_10_0

Reply:
GitHub uses [tag names](https://github.com/knik0/faad2/tags) for archives. The old repository on Sourceforge is abandoned.

Reply:
@pmetzger Any further concerns?